lib/w_module.dart

```diff
@@ -23,7 +23,8 @@ library w_module;
 
 export 'package:w_module/src/event.dart';
 export 'package:w_module/src/events_collection.dart';
-export 'package:w_module/src/lifecycle_module.dart' hide LifecycleState;
+export 'package:w_module/src/lifecycle_module.dart'
+    hide LifecycleState, maxChildUnloadDuration;
```

Comment:
Question: I'm surprised you're exporting maxChildUnloadDuration. Are you thinking consumers might want to adjust it in their unit tests?

Reply:
I'm not, this package has been using `hide` instead of `show` so I've added it to the `hide` list

Reply:
😆 I didn't see the word `hide`.